src/applications/disability-benefits/2346/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import Scroll from 'react-scroll';
+import { focusElement } from 'platform/utilities/ui';
+
+const scroller = Scroll.scroller;
+const scrollToTop = () => {
+  scroller.scrollTo('topScrollElement', {
+    duration: 500,
+    delay: 0,
+    smooth: true,
+  });
+};
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('.schemaform-title > h1');
+    scrollToTop();
+  }
+
+  render() {
+    return (
+      <>
+        <h2>
+          Your order is confirmed
+          <i className="form2346 fas fa-check-circle fa-4x" />
+        </h2>
+        <section>
+          <h2>What happens next</h2>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Temporibus
+            ex, repellendus est sed hic totam perspiciatis aut autem iusto earum
+            obcaecati dolorum cumque exercitationem distinctio fugit voluptas
+            commodi impedit voluptatibus.
+          </p>
```

Comment:
Did the design not give you proper text for this?

Reply:
We are not complete with the confirmation page yet. This is a work in progress because we have more pages to add to this form thus there will be more data that will be displayed on the confirmation page.